ios/Podfile

```diff
@@ -3,6 +3,8 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 
 target 'COVIDSafePaths' do
   # Pods for COVIDSafePaths
+  project 'COVIDSafePaths.xcodeproj'
```

Comment:
only needed if you have old PrivateKit artifacts under `ios/`

But probably work committing.

Reply:
This is from a merge, not intentional commit on my part

Reply:
its fine to commit. it helps when people have the old PrivateKit files left in their workdir.